examples/room/lod/lodmanager/manager.go

```diff
@@ -0,0 +1,207 @@
+// © 2019-2025 Diarkis Inc. All rights reserved.
+
+package lodmanager
+
+import (
+	"fmt"
+	"maps"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/Diarkis/diarkis/packet"
+	"github.com/Diarkis/diarkis/user"
+)
+
+// sendMessage represents a message to be sent to a user
+type sendMessage struct {
+	receiverUserID string
+	ver            uint8
+	cmd            uint16
+	payload        []byte
+}
+
+// Manager manages LOD (Level of Detail) synchronization for users
+type Manager struct {
+	userEntities          map[string]*UserEntity
+	started               *atomic.Bool
```

Comment:
```suggestion
	started               atomic.Bool
```
There is no reason to make `started` a pointer to atomic.Bool

Reply:
fixed